src/MarkdownTextInput.web.tsx

```diff
@@ -65,11 +65,6 @@ interface MarkdownNativeEvent extends Event {
   inputType: string;
 }
 
-type Selection = {
```

Comment:
I don't like that the Selection type was moved to parserUtils. It's mostly used inside the web component. How about moving it to a global type .d.ts file or cursorUtils?